src/adapters/deployer/index.ts

```diff
@@ -19,66 +19,68 @@ export function createDeployerComponent(
       try {
         const exists = await components.storage.exist(entity.entityId)
 
-        if (!exists) {
-          await components.downloadQueue.onSizeLessThan(1000)
+        const isSnsEntityToSend =
+          !!(entity.entityType === 'scene' || entity.entityType === 'wearable' || entity.entityType === 'emote') &&
+          components.sns.arn
+        const isSnsEventToSend = !!components.sns.eventArn
+        if (exists || !(isSnsEntityToSend && isSnsEventToSend)) {
```

Comment:
Do you think this could be easier to read?
```suggestion
        if (exists || !isSnsEntityToSend || !isSnsEventToSend) {
```

Reply:
We need to specify that both `isSnsEntityToSend` and `isSnsEventToSend` are false

Reply:
Using `!(isSnsEntityToSend && isSnsEventToSend)`, if any of `isSnsEntityToSend` or `isSnsEventToSend` are false, then `!(isSnsEntityToSend && isSnsEventToSend)` will be true. If both of them must be false then I think we should use ``(!isSnsEntityToSend && !isSnsEventToSend)`. WDYT?